Error when changing filter

stash-logger.js:114 Error: <web-directory@component:device-selection::ember6005> had no action handler for: (action (mut selectedFilter))

This doesn't appear to cause problems so I was wondering if we could just remove the mut handler.

--- a/addon/components/device-selection/component.js
+++ b/addon/components/device-selection/component.js
@@ -116,6 +116,10 @@ export default Component.extend(/* LoggerMixin, */{
       if (this.get('selectedResolution.presetId') !== id) {
         this.set('selectedResolution', this.get('webrtc.resolutionList').findBy('presetId', id));
       }
+    },
+
+    changeFilter (filter) {
+      this.set('selectedFilter', filter);
     }
   }
 });

--- a/addon/components/device-selection/template.hbs
+++ b/addon/components/device-selection/template.hbs
@@ -36,10 +36,10 @@
                         {{#x-select
                            id=(concat elementId '-filter-select')
                            value=selectedFilter
-                           action="(action (mut selectedFilter))"
+                           action="changeFilter"
                            class="form-control filter-select"}}
-                            {{#each advancedOptions as |option|}}
-                                {{#x-option value=option}}{{option}}{{/x-option}}
+                            {{#each advancedOptions as |filter|}}
+                                {{#x-option value=filter}}{{filter}}{{/x-option}}
                             {{/each}}
                         {{/x-select}}
                     </div>

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ember-webrtc-devices",
-  "version": "1.0.2",
+  "version": "1.0.3",
   "description": "Enumerating and selecting media devices from the browser as an ember addon",
   "directories": {
     "doc": "doc",

